cni/pkg/ipset/nldeps_windows.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Istio Authors
```

Comment:
does this need `//go:build !windows`?

Reply:
Nope, the _windows piece tells Go that this is only for windows